evm/src/main/java/org/hyperledger/besu/evm/processor/MessageCallProcessor.java

```diff
@@ -153,7 +153,7 @@ private void executePrecompile(
       final Bytes output = contract.compute(frame.getInputData(), frame);
       operationTracer.tracePrecompileCall(frame, gasRequirement, output);
       if (output != null) {
-        if (contract.getName().equals("Privacy") || contract.getName().equals("OnchainPrivacy")) {
+        if (contract.getName().equals("Privacy") || contract.getName().equals("FlexiblePrivacy")) {
```

Comment:
I think maybe this is a breaking change too? @pinges ? Maybe we need to check for both

Reply:
8d6d5f4